clients/da-vinci-client/src/main/java/com/linkedin/davinci/ingestion/isolated/IsolatedIngestionServer.java

```diff
@@ -203,6 +207,7 @@ public boolean startInner() {
     LOGGER.info("All ingestion components are initialized.");
 
     heartbeatCheckScheduler.scheduleAtFixedRate(this::checkHeartbeatTimeout, 0, 5, TimeUnit.SECONDS);
+    metricsCollectionScheduler.scheduleAtFixedRate(this::reportMetricsUpdateToMainProcess, 0, 10, TimeUnit.SECONDS);
```

Comment:
Even if this is true, since each collection will take a bit of time to complete, in the main process we are not seeing the last metric data. If we slow down the metric retrieval to 1 per minute, then will it be too stale? I mean the data could be more than 1 minutes old in extreme case

Reply:
Okay it seems like metric will be at most 1 minute stale. But is that desired?